templates/block/block--nidirect-article-teasers-by-topic.html.twig

```diff
@@ -0,0 +1,39 @@
+{#
+/**
+ * @file
+ * Theme override to display a block.
+ *
+ * Available variables:
+ * - plugin_id: The ID of the block implementation.
+ * - label: The configured label of the block if visible.
+ * - configuration: A list of the block's configuration values.
+ *   - label: The configured label for the block.
+ *   - label_display: The display settings for the label.
+ *   - provider: The module or other provider that provided this block plugin.
+ *   - Block plugin specific settings will also be stored here.
+ * - content: The content of this block.
+ * - attributes: array of HTML attributes populated by modules, intended to
+ *   be added to the main container tag of this template.
+ *   - id: A valid HTML ID and guaranteed unique.
+ * - title_attributes: Same as attributes, except applied to the main title
+ *   tag that appears in the template.
+ * - title_prefix: Additional output populated by modules, intended to be
+ *   displayed in front of the main title tag that appears in the template.
+ * - title_suffix: Additional output populated by modules, intended to be
+ *   displayed after the main title tag that appears in the template.
+ *
+ * @see template_preprocess_block()
+ */
+#}
+<div{{ attributes }}>
+  {% block content %}
+    {% for article in content.nidirect_article_teasers_by_topic %}
+      <div>
+        <h2>{{ article.title_link }}</h2>
+        <div>{{ article.summary_text }}{{ article.more_link }}</div>
+      </div>
+    {% endfor %}
+    {# Bubble cache tags up #}
+    {%  set catch_cache = content.nidirect_article_teasers_by_topic|render %}
```

Comment:
Haven't seen this trick before, nice one. Usually I just do the `render content | without` filter but it can get a bit awkward.